staging/src/k8s.io/kubectl/pkg/drain/filters.go

```diff
@@ -256,3 +256,12 @@ func (d *Helper) skipDeletedFilter(pod corev1.Pod) PodDeleteStatus {
 	}
 	return MakePodDeleteStatusOkay()
 }
+
+func (d *Helper) isWorkloadDaemonSet(workloadRef *metav1.OwnerReference) bool {
+	// find if workloadRef is daemonSet
+	daemonSetAPIVersion, daemonSetKind := appsv1.SchemeGroupVersion.WithKind("DaemonSet").ToAPIVersionAndKind()
+	if workloadRef.Kind == daemonSetKind && workloadRef.APIVersion == daemonSetAPIVersion {
```

Comment:
we can collapse this if statement into ```return workloadRef...``` here.

Reply:
fixed